# Strange Python Bug - Stagger (ID3) encoding errors from cron, but not from command line
LANG
